src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingWithCustomSignatureMethod.cs

```diff
@@ -22,6 +22,10 @@ public class SigningAndVerifyingWithCustomSignatureMethod
 <test>some text node</test>
 </example>";
 
+        private static bool SupportsSha2Algorithms =>
+            !PlatformDetection.IsFullFramework ||
+            CryptoConfig.CreateFromName("http://www.w3.org/2001/04/xmldsig-more#rsa-sha384") as SignatureDescription != null;
```

Comment:
CryptoConfig.CreateFromName("http://www.w3.org/2001/04/xmldsig-more#rsa-sha384") as SignatureDescription should be null in this case